community/concepts/grain.md

```diff
@@ -0,0 +1,59 @@
+### Grain
+
+A big part of SourceCred’s vision is the desire to create project-specific cryptoassets, called "Grain" that represent contribution to or support of a project. These cryptoassets hold promise to make open-source projects financially sustainable which is currently an unsolved problem. You can read more about grain [here](concepts/grain.md).
+
+#### Properties of Grain
+
+ - Grain is always project-specific. For example, "SourceCred grain" and "Maker Community grain" would be wholly different assets.
```

Comment:
Somewhat wary of using Maker as an example here, as they haven't committed to using SourceCred. Don't want to imply an endorsement. Though I suppose that's true of any crypto project at this point.